mlir/lib/Target/LLVMIR/Dialect/OpenMP/OpenMPToLLVMIRTranslation.cpp

```diff
@@ -3786,34 +3830,13 @@ static void initTargetDefaultBounds(
 
   // Calculate reduction data size, limited to single reduction variable
   // for now.
+  // FIXME: This treats 'DO SIMD' as if it was a 'DO' construct. Reductions
+  // on other constructs apart from 'DO' aren't considered either.
```

Comment:
Nit: We can remove this comment now, since we're no longer looking for the wrappers of an `omp.loop_nest`.